packages/cli/src/util/handle-error.ts

```diff
@@ -3,6 +3,44 @@ import info from './output/info';
 import errorOutput from './output/error';
 import { APIError } from './errors-ts';
 import { getCommandName } from './pkg-name';
+import { type Output } from './output/create-output';
+
+export function printError(error: unknown, output: Output) {
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (soft-blocking):** can we make to use this somewhere in this PR or a quick follow up?

I don't like adding unused code because it can sometimes stay that way.